app/controllers/ApplicationController.scala

```diff
@@ -20,14 +26,14 @@ import scala.concurrent.Future
 class ApplicationController @Inject() (
   val messagesApi: MessagesApi,
   val env: Environment[User, JWTAuthenticator])
-  extends Silhouette[User, JWTAuthenticator] {
+  extends Silhouette[User, JWTAuthenticator] with AuthenticationController {
 
   /**
    * Returns the user.
    *
    * @return The result to display.
    */
-  def user = SecuredAction.async { implicit request =>
+  def user = SecuredAction(WithRole("Student")).async { implicit request =>
```

Comment:
IMHO "Student" and other role-String should be extracted into app/utils/silhouette/Authorization.scala (something like ```val ROLE_STUDENT = "Student"```) and then be referenced instead of a direct String. This avoids typos etc.